firefox-ios/Client/Assets/CC_Script/CC_Python_Update.py

```diff
@@ -37,7 +37,6 @@
     "toolkit/components/formautofill/Helpers.ios.mjs",
     "toolkit/components/formautofill/shared/HeuristicsRegExp.sys.mjs",
     "toolkit/components/formautofill/shared/LabelUtils.sys.mjs",
-    "toolkit/components/passwordmgr/LoginManager.shared.mjs",
```

Comment:
This is no longer needed. It was replaced a while back and doesn't exist anymore on central.